EEJT-000A/test/java/task2/InputValidationTest.java

```diff
@@ -0,0 +1,40 @@
+package task2;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static task2.InputValidation.*;
```

Comment:
Static imports are fine only for test classes (asserts, mocks etc.)
Please remove static import of `InputValidation`